packages/slice-machine/test/src/utils/screenshots/missing.test.ts

```diff
@@ -1,4 +1,4 @@
-import { countMissingScreenshots } from "@src/utils/screenshots/missing";
+import { countMissingScreenshots } from "@src/domain/Slice";
```

Comment:
⚠️ Let's remove this file, or move it to domain.test.ts to test where the actual function is now